modules/microshift-nw-enforcing-hsts-per-domain.adoc

```diff
@@ -34,7 +34,7 @@ $ oc annotate route --all --all-namespaces --overwrite=true "haproxy.router.open
 ----
 $ oc annotate route --all -n <my_namespace> --overwrite=true "haproxy.router.openshift.io/hsts_header"="max-age=31536000;preload;includeSubDomains" <1>
```

Comment:
add italics to my namespace
_<my_namespace_

Reply:
done